### DISCUSSION TITLE
Problem with Deployment of TailwindCss into a Fastapi app on deta micro

### DISCUSSION THREAD

**Discussion by opticrew** [fastapi/fastapi#5555] [Questions] [answered]
Do you know how to deploy tailwindcss in production for a fastapi app serving Jinja html templates on deta.sh ?

Everything works very well on my local computer thanks to the tailwindcss cli.

Once deployed on deta.sh, everything works except the tailwindcss style are not applied.

Can someone help me ?

Thanks in advance.

Antoine

**Comment by opticrew** [comment, marked as answer]
It worked, I had just to delete my navigation history in my browser ...

**Reply by tiangolo** [reply]
Thanks for reporting back!
